### PR TITLE
Add recipe for falcon

### PR DESCRIPTION
**Pull request by bebatut**
* [X] I have read the guidelines above.
* [X] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/falcon/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pypeflow
+    - networkx
+  run:
+    - python
+    - setuptools
```
Comment (bgruening) [inline recipes/falcon/meta.yaml]: setuptools is probably not needed

File: recipes/falcon/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{% set name = "falcon" %}
```
Comment (druvus) [inline recipes/falcon/meta.yaml]: It is a name collision with another falcon software in conda-forge.

Comment (bgruening) [inline recipes/falcon/meta.yaml]: Can we call it `pacbio_falcon`?

Comment (bebatut) [inline recipes/falcon/meta.yaml]: I changed thatn

**Pull request merged**